lib/measures/tbd/resources/psi.rb

```diff
@@ -1002,9 +1002,8 @@ def safe(id = "", type = nil)
       end
 
       unless @has[id][safer]
-        safer = :fenestration if safer.to_s.include?("head")
-        safer = :fenestration if safer.to_s.include?("sill")
-        safer = :fenestration if safer.to_s.include?("jamb")
+        safer = :fenestration if safer == :skylight
+        safer = :fenestration if safer == :door
```

Comment:
Updates _subsurface_ edge type inheritance rules, which are based on (built-in or user-provided) PSI factor set entries. Supports 90.1 _subsurface_ differentiation.